### ISSUE TITLE
Doesn't work on Windows 10 (1909)

### ISSUE THREAD

**Issue by johngladp** [p-ranav/indicators#55]
Hi,
Is this expected to work on Windows and Window 10?

Running on Windows 10 Pro: 10.0.18363 (1909) the contents of the progress bars contain lots of invalid characters:
![image](https://user-images.githubusercontent.com/25591355/82667848-8ca1d200-9c30-11ea-8c69-9475b686555a.png)

The binary was built using Visual Studio 2019 16.6.0, cmake folders build, vanilla configure with samples enabled.

**Comment by p-ranav** [comment]
Yes it is expected to work in Windows. The sample you are running, `multi_block_progress_bar`, has unicode characters. This requires some configuration in Windows. [See here](https://stackoverflow.com/questions/388490/how-to-use-unicode-characters-in-windows-command-line)

**Comment by bzctoons** [comment]
you can add     SetConsoleOutputCP(CP_UTF8) in the ProgressBar constructor

**Comment by MindStudioOfficial** [comment]
> you can add SetConsoleOutputCP(CP_UTF8) in the ProgressBar constructor

Can you show an example of this?
